### ISSUE TITLE
Library has imports broken

### ISSUE THREAD

**Issue by salvajavier** [neeraj3029/chi-sq-test#5]
Angular CLI: 13.3.0
Node: 16.13.1
Package Manager: npm 8.5.5
OS: linux x64

Importing the library and triyng to use it throws:

```
Compiled with problems:X

ERROR in ./node_modules/chi-sq-test/lib/index.js 3:0-47

Module not found: Error: Can't resolve './goodnessOfFitTest.js' in '/usr/src/app/node_modules/chi-sq-test/lib'


ERROR in ./node_modules/chi-sq-test/lib/index.js 4:0-55

Module not found: Error: Can't resolve './independenceTest.js' in '/usr/src/app/node_modules/chi-sq-test/lib'
```

**Comment by hasansultan92** [comment]
Can vouch, I am also having a similar issue

**Comment by neeraj3029** [comment]
This should be fixed with the version `2.1.1`

**Comment by neeraj3029** [comment]
Thanks for pointing this out @salvajavier @hasansultan92
